odom/config/odom_local.yaml

```diff
@@ -28,15 +28,6 @@ odom0_queue_size: 10
 odom0_differential: false
 odom0_relative: false
 
-odom1: zed/zed_node/odom
```

Comment:
Is the 2d mode and removal of zed as an Odom source intentional?

Reply:
We'll need it regardless if we aren't allowed to tilt the camera back from 50 deg. I have no idea if the camera is adjustable nor are we going to